src/HealthcareApis/HealthcareApis/HealthcareApis/RemoveAzureRmHealthcareApisService.cs

```diff
@@ -0,0 +1,114 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.HealthcareApis.Properties;
+using Microsoft.Azure.Commands.HealthcareApis.Common;
+using Microsoft.Azure.Commands.HealthcareApis.Models;
+using Microsoft.Azure.Management.HealthcareApis;
+using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.HealthcareApis.Commands
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HealthcareApisService", DefaultParameterSetName = ServiceNameParameterSet, SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveAzureRmHealthcareApisService : HealthcareApisBaseCmdlet
+    {
+        protected const string ServiceNameParameterSet = "ServiceNameParameterSet";
+        protected const string ResourceIdParameterSet = "ResourceIdParameterSet";
+        protected const string InputObjectParameterSet = "InputObjectParameterSet";
+
+        [Parameter(
+          ParameterSetName = ServiceNameParameterSet,
+          Mandatory = true,
+          ValueFromPipelineByPropertyName = true,
```

Comment:
@poadhika same comment about removing `ValueFromPipelineByPropertyName = true` from both `-Name` and `-ResourceGroupName`.